nginx/default.conf

```diff
@@ -0,0 +1,36 @@
+server {
+  listen 8080;
+
+  location /api/user-service {
+    proxy_pass http://user-service:8000/api/user;
+  }
+
+  location /api/matching-service/socket {
+    proxy_pass http://matching-service:8001;
+    proxy_http_version 1.1;
```

Comment:
Perhaps can add comment to explain these upgrade and 1.1 keepalive connection is for enabling WebSockets? Thanks 😄 Same for those below.

Reply:
Agree